### ISSUE TITLE
[Feature Request] Support for PEP 621

### ISSUE THREAD

**Issue by Lee-W** [commitizen-tools/commitizen#212]
## Description
[PEP 621](https://www.python.org/dev/peps/pep-0621/) define `version` under `project`.  e.g.,

```toml
[project]
name = "spam"
version = "2020.0.0"
description = "Lovely Spam! Wonderful Spam!"

```

Currently, we might define `version` for multiple tools. For example, if we use `poetry` and `commitizen` in the same project, we'll have to make sure we sync up these two values. Once this is supported by both `poetry`, `commitizen`  and even other tools. We no longer need to worry about syncing problems.

## Possible Solution
Use the `version` under `project` if it exists.

## Related Issue
If applicable, add link to existing issue also help us know better.

**Comment by Woile** [comment]
At the moment that can be done with `version_files`, you can have `pyproject.toml:version` and it will be updated automatically, so it's quite cheap to keep it in sync.

The disadvantage of this proposal is that there are many other 'settings' related to commitizen, so if we use `[project] version` it will be harder to maintain `[tool.commitizen]`, and you will have settings related to commitizen in different places.

Basically you are proposing to be able to skip `version` in `[tool.commitizen]` if I understand correctly, but that would apply only to python projects, I use it for JS and python, and by supporting `.cz.toml` you can use it anywhere.

**Comment by Lee-W** [comment]
Yes, that makes sense. Even although we won't add this feature, I think this discussion why we won't support it is worth mentioning in our documentation (maybe FAQ?)

**Comment by phil65** [comment]
Just a small tip that you should use `pyproject.toml:^version` in `version_files`, otherwise poetry dependencies and similar stuff might also accidentally get bumped.

**Comment by Woile** [comment]
I'll create a FAQ, I think it's a good idea :+1: 

Thanks @phil65 that's great advice :muscle:
